osc/commandline.py

```diff
@@ -2423,15 +2429,9 @@ def do_request(self, subcmd, opts, *args):
                 states = ('new')
                 results = get_request_collection(apiurl, project=project, package=package, states=['new'])
             else:
-                state_list = opts.state.split(',')
-                if state_list == ['']:
-                    state_list = ()
-                if opts.all:
-                    state_list = ['all']
-                else:
-                    for s in state_list:
-                        if s not in states and not s == 'all':
-                            raise oscerr.WrongArgs('Unknown state \'%s\', try one of %s' % (s, ','.join(states)))
+                for s in state_list:
+                    if s not in states and not s == 'all':
```

Comment:
NIT: I know it has been that way before

```suggestion
                    if s!= 'all' and s not in states:
```

Reply:
The proposed change makes sense to me - it's definitely an improvement to code readability. Thanks.